Source/WebKit/UIProcess/ios/WebPageProxyIOS.mm

```diff
@@ -547,7 +547,17 @@ static bool exceedsRenderTreeSizeSizeThreshold(uint64_t thresholdSize, uint64_t
     if (!hasRunningProcess())
         return callbackFunction(false, RevealItem());
 
-    sendWithAsyncReply(Messages::WebPage::PrepareSelectionForContextMenuWithLocationInView(point), WTFMove(callbackFunction));
+    dispatchAfterCurrentContextMenuEvent([weakThis = WeakPtr { *this }, point, callbackFunction = WTFMove(callbackFunction)] (bool handled) mutable {
+        if (!weakThis)
+            return;
```

Comment:
I believe we'd still want to invoke the completion handler in this case; otherwise, UIKit might get into a bad state.

Reply:
oh good catch